chat-plugins/hangman.js

```diff
@@ -225,7 +225,7 @@ exports.commands = {
 
 		guess: function (target, room, user) {
 			if (!target) return this.parse('/help guess');
-			if (!room.game || room.game.gameType !== 'hangman') return this.errorReply("There is no game of hangman running in this room.");
+			if (!room.game || room.game.title !== 'Hangman') return this.errorReply("There is no game of hangman running in this room.");
```

Comment:
Titles aren't IDs. @kotarou3's implementation of tours didn't work in groupchats because he assumed titles were IDs. A title can be anything the user/programmer wants it to be. Don't make the same mistake.